src/components/protection/WithdrawProtectionSingle.vue

```diff
@@ -188,6 +195,13 @@ export default class WithdrawProtectionSingle extends BaseComponent {
     return pos;
   }
 
+  get rewardsWithMultiplier() {
+    for (let position of vxm.ethBancor.protectedPositions)
+      if (position.rewardsMultiplier.gt(1)) return true;
+
+    return false;
```

Comment:
Tad bit verbose, lets just do something like
```suggestion
    return vxm.ethBancor.protectedPositions.some(position => position.rewardsMultiplier.gt(1))
```